src/Pane.tsx

```diff
@@ -30,7 +30,29 @@ export function Pane(props: PaneProps) {
       if(patchedStyle.flexBasis === 'auto' && basis) {
         patchedStyle.flexBasis = basis;
       }
-    }                                                                                         
+    } 
+    
+    if(state.split == 'vertical') {
+      if(state.minSize > 0) {
+        patchedStyle.minWidth = state.minSize;
```

Comment:
for some reason this one does not apply for div styles even if there is truly state.minSize appeared